doc/command_ref.rst

```diff
@@ -1759,6 +1763,53 @@ of the most recent transaction. The last form is ``last-<offset>``, where
 ``<offset>`` is a positive integer. It specifies offset-th transaction preceding
 the most recent transaction.
 
+.. _excluded_packages-label:
+
+=================
+Package filtering
```

Comment:
There should be a capital F here - english headings have caps on each word (except prepositions etc.):
> Package Filtering

(I've left the non-capital f in the first paragraph of the text below, I think it can be either way there, depending on how you look at it...)